### ISSUE TITLE
ctrl-6 does not switch to alternate file 

### ISSUE THREAD

**Issue by socketwiz** [onivim/oni#2002]
http://vimdoc.sourceforge.net/htmldoc/editing.html#CTRL-^

In vim proper (and I also tested in neovim) when you have multiple buffers open you can switch to the last one you had open with ctrl-6.  This does not seem to work in ONI.  I've even tried setting:

```
    "tabs.mode": "native"
```

How are people switching buffers without the use of the mouse in ONI?

**Comment by justinmk** [comment]
ctrl-6 only works by accident in some terminals. The actual builtin mapping is `<C-^>`.

**Comment by socketwiz** [comment]
> I've attempted to repro this as follows:

> :e file1
> :e file2
> <ctrl-6> to try swap back to file1.
> Does that seem sane?

Yes, exactly.

> It doesn't work for me on Oni/Neovim/Neovim-Qt, so I'm thinking its either a keyboard thing or a Windows thing.

It works for me with both vim and neovim, just not oni.  I do not use windows, I use OS X and Arch Linux.

> That said, using <Ctrl+Shift+6> does work for me, so I'm a little confused.

Yes, that seems to work for me as well.

> ctrl-6 only works by accident in some terminals. The actual builtin mapping is <C-^>.

That's not true.  According to the VIM documentation both are valid:

http://vimdoc.sourceforge.net/htmldoc/editing.html#CTRL-^

> \*CTRL-^\* \*CTRL-6\* Edit the alternate file.

This has worked for me for well over a decade everywhere I've ever used vim.  Heck it even works in emacs with evil-mode everywhere I've used emacs.

**Comment by justinmk** [comment]
> That's not true. According to the VIM documentation both are valid

That doesn't contradict what I said. ctrl-6 happens to send `ctrl-^` on some terminals. You may ask why ctrl-5 (or any other number besides 6) doesn't work (if you map it to something).

In Terminal.app on macOS 10.13, ctrl-6 produces nothing (at least on my machine).

Anyways, the point of my previous comment is that this is something Oni (and other implementations, such as evil-mode) would need to map explicitly (due to user expectation).

**Comment by justinmk** [comment]
In fact, even Vim itself does not actually support ctrl-6 as a builtin mapping. You can try it:

    :call feedkeys("\<c-6>")

That does not, whereas `ctrl-^` does switch the buffer:

    :call feedkeys("\<c-^>")

**Comment by socketwiz** [comment]
> In fact, even Vim itself does not actually support ctrl-6 as a builtin mapping.

I don't know, maybe I have special machines but on both OS X and and Arch Linux when I run vim proper without any configuration (`vim -u NONE`) ctrl-6 will switch back to previous buffer.

However I will concede.  I have found that the following configuration makes ONI work when I set it in my neovim config (`~/.config/nvim/init.vim`) which I have enabled in the oni config with `"oni.loadInitVim": true`.

```
noremap <C-6> <C-^>
```
